extracted/konan.serializer/src/org/jetbrains/kotlin/descriptors/konan/KonanModuleDescriptorUtils.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.kotlin.descriptors.konan
+
+import org.jetbrains.kotlin.descriptors.ModuleDescriptor
+import org.jetbrains.kotlin.konan.library.KONAN_STDLIB_NAME
+import org.jetbrains.kotlin.name.Name
+
+private val STDLIB_MODULE_NAME = Name.special("<$KONAN_STDLIB_NAME>")
+
+fun ModuleDescriptor.isKonanStdlib() = name == STDLIB_MODULE_NAME
```

Comment:
Btw, some time ago it appeared that our `isStdlib` predicate doesn't work properly. Have you checked this one?

Reply:
Where can I find `isStdlib` predicate right now to check it?

AFAIR `isKonanStdlib` is exactly the same predicate. It has been renamed from `isStdlib` some time ago (before this pull request) to avoid meaning clash with stdlib from Kotlin/JVM.

Reply:
Then it makes some sense to check whether code under `if (module. isKonanStdlib())` really gets executed.

Reply:
Discussed outside. There is no issue with this predicate.